tls/test_tls_cert.py

```diff
@@ -448,21 +447,21 @@ def start_all(self):
 
     def check_handshake_success(self, sni):
         """Run TLS handshake with the given SNI and check it is completes successfully."""
-        hs = TlsHandshake(verbose=self.verbose)
-        hs.sni = [sni]
+        hs = TlsHandshake()
+        hs.sni = sni
         # TLS 1.2 handshake completed with no exception => SNI is accepted
-        hs._do_12_hs()
-        self.assertTrue(x509_check_cn(hs.cert, "tempesta-tech.com"))
+        hs.do_12()
+        self.assertTrue(x509_check_cn(hs.hs.server_cert[0], "tempesta-tech.com"))
 
     def check_handshake_unrecognized_name(self, sni):
         """
         Run TLS handshake with the given SNI
         and check server name is not recognised by the server.
         """
-        hs = TlsHandshake(verbose=self.verbose)
```

Comment:
Why verbose output is removed?

Reply:
Cause it just was added to pass verbose into test
For now `-v -v -v ` flags turn verbose on